### PR TITLE
fixes #5854 - org creation non-admin normal user doesn't add the org name in org switcher

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by dmitri-d** [comment]
This also fixes similar issue with locations. Could you add a test for this?

**Comment by domcleal** [comment]
Would it be possible/better to make the user selected by default on instantiation?  e.g. the user will see themselves as in the org when on the Users tab, but then also have the option to remove themselves should they wish to make an org for other people.

**Comment by isratrade** [comment]
@domcleal, new org creation users the wizard, I added another commit (to squash) to go to step 3 (edit) if user is not admin

**Comment by domcleal** [comment]
Ok, ignore my above comment, I'd forgotten you can only create a taxonomy through the wizard, makes more sense now.

Just outstanding is the above request for a unit test.

**Comment by isratrade** [comment]
re-committed with unit tests added.

**Comment by domcleal** [comment]
Merged as a1af109 for Foreman 1.6.0, thanks @isratrade.

File: app/models/taxonomy.rb [hunk 1]
```diff
@@ -138,4 +139,9 @@ def hash_key_to_class(key)
     key.to_s.gsub(/_ids?\Z/, '').classify
   end
 
+  def assign_taxonomy_to_user
+    return if User.current.admin
+    TaxableTaxonomy.create(:taxonomy_id => self.id, :taxable_id => User.current.id, :taxable_type => 'User')
```
Comment (ohadlevy) [inline app/models/taxonomy.rb:144]: should we have a couple of extra checks here? for example:
- user already selected himself (no need to add)
- user is already in a group that has access?

**Pull request closed without merging**